<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(155, 221, 209);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#sign_in_button
{
border: 5px solid;
border-radius: 20%;
border-color: rgb(32, 37, 42);
background-color: rgb(28, 115, 100);
color: rgb(204, 196, 180);
}
QPushButton:hover#sign_in_button
{
   background-color:rgb(1,58,51);
}
QPushButton:pressed#sign_in_button
{
   background-color:gray;
}
QLineEdit{
	padding:10px;
}</string>
   </property>
   <widget class="QPushButton" name="sign_in_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>450</y>
      <width>360</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TT_Skip-E</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Sign in</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="login_input">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>380</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TT_Skip-E</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid;
border-radius: 20%;
border-color: rgb(32, 37, 42);
background-color: rgb(204, 196, 180);
color: rgb(44, 132, 116);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_input">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>380</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TT_Skip-E</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid;
border-radius: 20%;
border-color: rgb(32, 37, 42);
background-color: rgb(204, 196, 180);
color: rgb(44, 132, 116);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TT_Skip-E</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TT_Skip-E</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>60</y>
      <width>170</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TT_Skip-E</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <zorder>sign_in_button</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>password_input</zorder>
   <zorder>login_input</zorder>
   <zorder>label_3</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
